QUARKUS-3461.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-3461 Verify behaviour and functionality of Quarkus CLI update command with RHBQ
+- JIRA link: https://issues.redhat.com/browse/QUARKUS-3461
+- Quarkus Guides:
+    - https://quarkus.io/guides/update-quarkus
+
+## Scope of the testing
+- Exploratory testing of the feature over Quarkus upstream and last released version of RHBQ
+- Verify that the `quarkus update` command explicitly warns users about deprecated/unsupported extensions during migration to the newer version.
+  In most cases deprecated extensions are moved to `io.quarkiverse`, which is accepted for Quarkus upstream but not for RHBQ.
+- Test coverage in QQE TS:
+  - Create a Quarkus/RHBQ app with the oldest possible stream and update to the latest version - verify new Quarkus platform version and app functionality
+
+## Getting familiar with the feature
+Following actions were taken to ensure familiarity:
+- Ensure documentation provides clear explanation on configuration options
+- Ensure good user experience and simplicity of use
+
+## Existing test coverage
+- [Quarkus updates](https://github.com/quarkusio/quarkus-updates/tree/main/recipes-tests/src/test) 
```

Comment:
Camel Quarkus team at the very least explored this in past, I think they had some requirement to test this command, not sure if they succeeded. Did you checked for their test coverage or contacted them? I think it would be interesting to know what they are doing and how.